src/style.css

```diff
@@ -0,0 +1,157 @@
+:root {
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+  --background-button-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  padding: 32px 19px 40px;
+}
+
+.card__title {
+  font-family: "Roboto", sans-serif;
```

Comment:
you don't need to write font-family in selectors, you already added font in html for all elements, delete in all selectors except html